example/lib/main.dart

```diff
@@ -28,6 +59,8 @@ void main() {
 class HomePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
+    AEPMessaging.Messaging.setMessagingDelegate(CustomMessagingDelegate());
```

Comment:
We were doing that because this is the entry point of the app which we want the in-app message to show when launching.  
After the testing,we know this works.  @dsoffiantini we can move this back to messaging.dart.  User and us can test with track calls for the in app instead.